### PR TITLE
Clarify team* freeform fields in jury interface

### PR DESCRIPTION
**Pull request by vmcj**
This is the first of the 2 items mentioned by Nicky in https://github.com/DOMjudge/domjudge/issues/872

Mainly to clarify the difference in intended usage between the 2 fields.

### CODE REVIEW

File: webapp/config/load_db_secrets.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ function get_db_url(): string
         break;
     }
 
-    return sprintf('mysql://%s:%s@%s:%d/%s?serverVersion=5.7.0', $user, $pass, $host, $port ?? 3306, $db);
+    return sprintf('mysql://%s:%s@%s:%d/%s?serverVersion=mariadb-10.7.3', $user, $pass, $host, $port ?? 3306, $db);
```
Comment (nickygerritsen) [inline webapp/config/load_db_secrets.php:37]: Do we want this?

File: webapp/templates/jury/team.html.twig [hunk 2]
```diff
@@ -150,17 +150,17 @@
...
-                <tr>
-                    <th>Comments</th>
-                    <td>
-                        {{ team.comments | nl2br }}
-                    </td>
-                </tr>
+                    <tr>
+                        <th>Comments</th>
+                        <td>
+                            {{ team.comments | nl2br }}
```
Comment (nickygerritsen) [inline webapp/templates/jury/team.html.twig:156]:
```suggestion
                            {{ team.internalComments | nl2br }}
```

Right?

**Review by nickygerritsen — COMMENTED** [review]
The name of this PR is not really correct, it has nothing to do with members (anymore)

File: webapp/config/load_db_secrets.php (hunk 1 above)
Comment (vmcj) [reply webapp/config/load_db_secrets.php:37]: It was for me to check if that would make it easier to generate doctrine:migrations:diff.

But I still needed to manually fix the migrations so the commit message already reminds me to later remove it.

File: webapp/templates/jury/team.html.twig (hunk 2 above)
Comment (vmcj) [reply webapp/templates/jury/team.html.twig:156]: Yes

File: webapp/config/load_db_secrets.php (hunk 1 above)
Comment (nickygerritsen) [reply webapp/config/load_db_secrets.php:37]: I have a similar thing on my Linux machine but think I didn't have it on my mac with Docker (even after adding stuff to `.env.local`). We might want to investigate later.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**